dockerfiles/testsuite/js/gen-vote-snark.js

```diff
@@ -31,16 +31,24 @@ async function generateProof(inputs, path) {
   globalThis.curve_bn128.terminate();
 }
 
-async function downloadFile(id, url, path, filename) {
+async function downloadFile(id, url, path, hashB64, filename) {
   // console.log("download file:", filename);
-  const file = fs.createWriteStream(`${path}/${id}/${filename}`);
+  const hashHex = Buffer.from(hashB64, "base64").toString("hex");
```

Comment:
Maybe it's better to do the downloading & hash verification from go using 'zk artifacts' package: https://github.com/vocdoni/vocdoni-node/blob/master/crypto/zk/artifacts/artifacts.go#L76
As eventually in the future the proof verification will also be computed from go code, so the js script will no be longer used. So I think that makes sense to currently download & verify hashes from go.